indy_client/test/cli/test_send_nym_validation.py

```diff
@@ -1,541 +0,0 @@
-import pytest
-from libnacl import randombytes
-
-from plenum.common.util import rawToFriendly, friendlyToHexStr, friendlyToHex, \
-    hexToFriendly
-from indy_client.test.cli.constants import ERROR, INVALID_SYNTAX
-from indy_client.test.cli.helper import createUuidIdentifier, \
-    createHalfKeyIdentifierAndAbbrevVerkey, createUuidIdentifierAndFullVerkey, \
-    createCryptonym
-from indy_common.roles import Roles
-
-NYM_ADDED = 'Nym {dest} added'
-
-
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
integrated